tests/markdown/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -1693,8 +1693,8 @@ Built-in parsers:
 
 [Custom parsers](#custom-parser-api) are also supported. _Since v1.5.0_
 
-| Default   | CLI Override                                    | API Override                                               |
-| --------- | ----------------------------------------------- | ---------------------------------------------------------- |
+| Default | CLI Override | API Override |
+| ------- | ------------ | ------------ |
 | \`babylon\` | \`--parser <string>\`<br />\`--parser ./my-parser\` | \`parser: "<string>"\`<br />\`parser: require("./my-parser")\` |
```

Comment:
Is this change intentional?

Reply:
Yeah, since there's a `html` and it's indeed very long.

Reply:
Looks like the first column got shorter even though it doesn’t have HTML in it.

Reply:
That's expected, isn't it? There is no other column with would set the wider char count that would warrant that whitespace?!?

Reply:
Since the table is row-based, we're not able to align only one column, though in this case it looks like we can but it's because it's the first column.

Reply:
Now I understand the logic here. LGTM 👍